index.js

```diff
@@ -56,3 +56,26 @@ module.exports.restore = function (store) {
 		cb();
 	});
 };
+
+module.exports.clear = function () {
+	if (arguments.length) {
+		var stores = Array.prototype.slice.call(arguments).filter(function(store) {
+			if (typeof store !== 'string') {
+				gutil.log('gulp-save', gutil.colors.red('Store names must be strings'));
+				return false;
+			}
+
+			return true;
+		});
+	}
+
+	return through.obj(function (file, enc, cb) {
+		cb();
```

Comment:
Sorry, you're correct.
I've lost my changes since yesterday and I rushed the code again today, but I missed that bit.
I was using `this.push(file)` indeed.